src/main/scala/codacy/duplication/pmd/Cpd.scala

```diff
@@ -1,22 +1,23 @@
 package codacy.duplication.pmd
 
-import java.io.{OutputStream, PrintStream}
+import java.io.{ByteArrayOutputStream, OutputStream, PrintStream}
+import java.nio.charset.StandardCharsets
 import java.nio.file.{Path, Paths}
 
 import codacy.dockerApi.api.{DuplicationClone, DuplicationCloneFile, DuplicationConfiguration, Language}
 import codacy.dockerApi.traits.IDuplicationImpl
 import net.sourceforge.pmd.cpd._
 
 import scala.collection.JavaConversions._
-import scala.util.Try
+import scala.util.{Failure, Success, Try}
 
 object Cpd extends IDuplicationImpl {
   override def apply(path: Path, config: DuplicationConfiguration): Try[List[DuplicationClone]] = {
-    val outStream = Console.out
-    val errStream = Console.err
+    val baos = new ByteArrayOutputStream()
+    val errStream = new PrintStream(baos, true, "utf-8")
 
-    System.setOut(NullPrintStream)
-    System.setErr(NullPrintStream)
+    System.setOut(errStream)
```

Comment:
Shouldn't this be "baos" instead of "errStream"?

Reply:
It needs a PrintStream. I just need baos to get the output from that PrintStream.